config/initializers/secure_headers.rb

```diff
@@ -10,6 +10,10 @@
     config.x_content_type_options = "nosniff"
     # X-XSS-Protection
     # X-Permitted-Cross-Domain-Policies
+
+    #FYI, this was deprecated and disabled in rails 7.  Using content security policy is the desired behavior going forward:
+    # https://github.com/rails/rails/commit/1f4714c3f798df227222f531141880b8e1b4170a
+    # https://github.com/rails/rails/blob/d437ae311f1b9dc40b442e40eb602e020cec4e49/railties/lib/rails/application/configuration.rb#L227
```

Comment:
FYI, I dropped the change to disable x-xss protection. It's a TODO to go back to the rails 7.1 defaults above.

Reply:
Added comment re: usage of unsafe-line requires we continue to use this for now.